pkg/ccl/crosscluster/physical/standby_read_ts_poller_job_test.go

```diff
@@ -54,23 +51,28 @@ func TestStandbyReadTSPollerJob(t *testing.T) {
 	readerTenantName := fmt.Sprintf("%s-readonly", args.DestTenantName)
 	c.ConnectToReaderTenant(ctx, readerTenantID, readerTenantName, 0)
 
+	c.SrcTenantSQL.Exec(t, `
+USE defaultdb;
+CREATE TABLE a (i INT PRIMARY KEY);
+INSERT INTO a VALUES (1);
+`)
+
+	srcTime = c.SrcCluster.Server(0).Clock().Now()
+	c.WaitUntilReplicatedTime(srcTime, jobspb.JobID(ingestionJobID))
+
 	pollInterval := time.Microsecond
 	serverutils.SetClusterSetting(t, c.DestCluster, "bulkio.stream_ingestion.standby_read_ts_poll_interval", pollInterval)
 
+	waitForPollerJobToStart(t, c)
+
 	var jobID jobspb.JobID
 	c.ReaderTenantSQL.QueryRow(t, `
```

Comment:
could we push this logic around waiting for the poller job to run into your `waitForPollerJobToStart` helper, and maybe rename it to `waitForPollerToRun`?